src/__tests__/mockdelete.test.js

```diff
@@ -1,5 +1,7 @@
 const removeAll = require('../__mocks__/mockdelete.js');
 
-test('Removing according to dummyArr 2 false status did not work', () => {
-  expect(removeAll()).toBe(1);
+describe('Test (All)removing functions', () => {
+  test('Removing according to dummyArr 2 false status did not work', () => {
+    expect(removeAll()).toBe(1);
+  });
 });
```

Comment:
You've done a really nice job with your tests and your code is easily understandable. Good job. :clap: 

Let's take a look at the following required changes in order to make it even better.

- [ ] Please clean up the commented codes you have inside `mockdeleteSingle.test.js`.
- [ ] Kindly include test for the function for editing the task description.

[OPTIONAL]

- [ ] Please consider adding details of your test to the README and how to execute the test.

Reply:
Hi sir, 
we already did the second change you want from us the file name is updater.test.js.
Hope you will clear exactly what do you want from us, please?
Thanks
@george-shammar